analyzers/tests/SonarAnalyzer.CSharp.Styling.Test/TestCases/UseRegexSafeIsMatch.cs

```diff
@@ -0,0 +1,63 @@
+﻿using System.Text.RegularExpressions;
+using RealRegex = System.Text.RegularExpressions.Regex;
+
+class UseRegexSafeIsMatchNonCompliant
+{
+    private Regex regex;
+    private RealRegex realRegex;
+
+    void InstanceRegex(string content)
+    {
+        regex.IsMatch(content);                                                // Noncompliant {{Use 'SafeIsMatch' instead.}}
+        //    ^^^^^^^
+        regex.IsMatch(content, 0);                                             // Noncompliant
+        regex.Matches(content);                                                // Noncompliant {{Use 'SafeMatches' instead.}}
+        regex.Matches(content, 0);                                             // Noncompliant
+        regex.Match(content);                                                  // Noncompliant {{Use 'SafeMatch' instead.}}
+        regex.Match(content, 0);                                               // Noncompliant
+        realRegex.IsMatch(content);                                            // Noncompliant
+        realRegex.IsMatch(content, 0);                                         // Noncompliant
+        realRegex.Matches(content);                                            // Noncompliant
+        realRegex.Matches(content, 0);                                         // Noncompliant
+        realRegex.Match(content);                                              // Noncompliant
+        realRegex.Match(content, 0);                                           // Noncompliant
```

Comment:
This first block has the comments bit too far. As long as they are all aligned, you can move them even closer than the second block below